android/build.gradle

```diff
@@ -15,21 +15,25 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', 26)
-    buildToolsVersion safeExtGet('buildToolsVersion', '26.0.3')
+    compileSdkVersion safeExtGet('compileSdkVersion', 28)
+    buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')
 
     defaultConfig {
-        minSdkVersion safeExtGet('minSdkVersion', 16)
-        targetSdkVersion safeExtGet('targetSdkVersion', 26)
+        minSdkVersion safeExtGet('minSdkVersion', 23)
```

Comment:
Can we increase the minSdkVersion?

Reply:
I copied the version numbers from the react-native-voice repo, let me know which versions it should be instead and I'll change it

Reply:
I'm wondering if we get incompatibility issues with AmiGO, where the minSdkVersion is 21. But that means the same is true for rnv...

Reply:
Shall l change it to 21 then to be sure?

Reply:
If there is no specific reason to use 23, then I guess that is the best.

Reply:
ok I changed it to 21